data.lua

```diff
@@ -0,0 +1,4 @@
+
+--- make data stage functions available globally
+---@usage flib.copy_prototype()
+flib = require("data.utilities")
```

Comment:
As per the bit on modules, please don't add things to the global namespace. Modules should be required locally in the files where they are used:

`local utilities = require("__flib__.data.utilities")`

Reply:
moved functions into table